<!DOCTYPE html>
<html lang="en-us" 
      xmlns:og="http://ogp.me/ns#" 
      xmlns:fb="https://www.facebook.com/2008/fbml">

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.8.0">

  

  
  

  
  
  
  
  
    
    
    
  
  

  

  
  
  
    
  
  <meta name="description" content="Biostatistical computing using R">

  
  <link rel="alternate" hreflang="en-us" href="/reading/">

  


  
  
  
  <meta name="theme-color" content="#456355">
  

  
  
  
  <script src="/js/mathjax-config.js"></script>
  

  
  
  
  
    
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/styles/github.min.css" crossorigin="anonymous" title="hl-light">
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" disabled>
        
      
    

    

    

    
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.1.2/lazysizes.min.js" integrity="sha256-Md1qLToewPeKjfAHU1zyPwOutccPAm5tahnaw7Osw0A=" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js" integrity="" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    

  

  
  
  
    
      
      
      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Fira+Sans+Condensed:400,400i,700,700i%7COverpass:400,400i,700,700i&display=swap">
    
  

  
  
  
  
  <link rel="stylesheet" href="/css/academic.css">

  




  


  
  

  
  <link rel="alternate" href="/reading/index.xml" type="application/rss+xml" title="Biostatistical Computing with R">
  

  <link rel="manifest" href="/index.webmanifest">
  <link rel="icon" type="image/png" href="/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_32x32_fill_lanczos_center_2.png">
  <link rel="apple-touch-icon" type="image/png" href="/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_192x192_fill_lanczos_center_2.png">

  <link rel="canonical" href="/reading/">

  
  
  
  
  
  
  
    
  
  
  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@mikhaildozmorov">
  <meta property="twitter:creator" content="@mikhaildozmorov">
  
  <meta property="og:site_name" content="Biostatistical Computing with R">
  <meta property="og:url" content="/reading/">
  <meta property="og:title" content="References | Biostatistical Computing with R">
  <meta property="og:description" content="Biostatistical computing using R"><meta property="og:image" content="/media/social-image.png">
  <meta property="twitter:image" content="/media/social-image.png"><meta property="og:locale" content="en-us">
  
    
  

  




  


  <link rel="shortcut icon" href="/favicon.ico" />
  <link rel="apple-touch-icon-precomposed" sizes="57x57" href="/img/apple-touch-icon-57x57.png" />
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="/img/apple-touch-icon-114x114.png" />
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="/img/apple-touch-icon-72x72.png" />
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/img/apple-touch-icon-144x144.png" />
  <link rel="apple-touch-icon-precomposed" sizes="120x120" href="/img/apple-touch-icon-120x120.png" />
  <link rel="apple-touch-icon-precomposed" sizes="152x152" href="/img/apple-touch-icon-152x152.png" />
  <link rel="icon" type="image/png" href="/img/favicon-32x32.png" sizes="32x32" />
  <link rel="icon" type="image/png" href="/img/favicon-16x16.png" sizes="16x16" />
  <meta name="application-name" content="BIOS 524: Biostatistical Computing with R" />
  <meta name="msapplication-TileColor" content="#FFFFFF" />
  <meta name="msapplication-TileImage" content="/img/mstile-144x144.png" />


  <title>References | Biostatistical Computing with R</title>

</head>


<body id="top" data-spy="scroll" data-offset="70"
    data-target="#TableOfContents"
    >

    <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


    












<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
    <div class="d-none d-lg-inline-flex">
      <a class="navbar-brand" href="/">Biostatistical Computing with R</a>
    </div>
    

    
    <button type="button" class="navbar-toggler" data-toggle="collapse"
            data-target="#navbar-content" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span><i class="fas fa-bars"></i></span>
    </button>
    

    
    <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
      <a class="navbar-brand" href="/">Biostatistical Computing with R</a>
    </div>
    

    
    
    <div class="navbar-collapse main-menu-item collapse justify-content-end" id="navbar-content">

      
      <ul class="navbar-nav d-md-inline-flex">
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/syllabus/"><span>Syllabus</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/schedule/"><span>Schedule</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link  active" href="/reading/"><span>Readings</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/assignment/"><span>Assignments</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/class/"><span>Classes</span></a>
        </li>

        
        

        

        
        
        
          
            
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="https://rstudio.cloud/" target="_blank" rel="noopener"><span>RStudio.cloud</span></a>
        </li>

        
        

      

        
      </ul>
    </div>

    <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
      

      

      

    </ul>

  </div>
</nav>



    

<div class="container-fluid docs">
  <div class="row flex-xl-nowrap">
    <div class="col-12 col-md-3 col-xl-2 docs-sidebar">
      





  
    
  




<form class="docs-search d-flex align-items-center">
  <button class="btn docs-toggle d-md-none p-0 mr-3" type="button" data-toggle="collapse" data-target="#docs-nav" aria-controls="docs-nav" aria-expanded="false" aria-label="Toggle section navigation">
    <span><i class="fas fa-bars"></i></span>
  </button>

  
</form>

<nav class="collapse docs-links" id="docs-nav">
  

  
  
  
  
  <div class="docs-toc-item">
    <a class="docs-toc-link" href="/reading/">Reading details</a>
    <ul class="nav docs-sidenav">
      
      <li class="active">
        <a href="/reading/">References</a>
      </li>
      
    </ul>
    

  </div>
  
  
</nav>

    </div>

    

    <main class="col-12 col-md-9 col-xl-8 py-md-3 pl-md-5 docs-content" role="main">

      <article class="article">

        <div class="docs-article-container">
          <h1>References</h1>

          <div class="article-style">
            
<script src="/rmarkdown-libs/header-attrs/header-attrs.js"></script>


<ul>
<li><p>Class-specific reading assignments will be posted under the <a href="/assignment/">Assignments</a> section following each class session</p></li>
<li><p>See <a href="https://github.com/mdozmorov/R_notes">“Data science in R notes”</a> GitHub repository for more resources</p></li>
</ul>
<div id="r" class="section level2">
<h2>R</h2>
<ul>
<li><p><a href="https://cran.r-project.org/doc/contrib/Torfs+Brauer-Short-R-Intro.pdf">A very short R introduction</a></p></li>
<li><p><a href="https://bigbookofr.netlify.app/">Big Book of R</a>, by Oscar Baruffa. A bookdown bookmarking links to everything R for data science, from basics, visualization, tidyverse to distributed computing, spatial data science, machine learning, and more. <a href="https://github.com/oscarbaruffa/BigBookofR">GitHub</a></p></li>
<li><p><a href="http://www.analyticsvidhya.com/blog/2016/02/complete-tutorial-learn-data-science-scratch/">A Complete Tutorial to learn Data Science in R from Scratch</a> - Basics of R, with short and conscise examples</p></li>
<li><p><a href="https://github.com/coatless/teach-r">teach-r</a> - List of Resources for Teaching R</p></li>
<li><p><a href="https://github.com/rstudio-education/rstats-ed">List of courses teaching R</a></p></li>
<li><p><a href="https://github.com/rstudio-education/primers">The learnr tutorials in RStudio Cloud primers</a>, avaliable on <a href="https://rstudio.cloud/projects">RStudio Cloud</a></p></li>
<li><p><a href="http://swirlstats.com/">swirl</a> - R package for interactive R leaning</p></li>
<li><p><a href="https://github.com/matloff/fasteR">FasteR</a> - Fast Lane to Learning R, learn R language in R console by following examples, by Norm Matloff</p></li>
<li><p><a href="https://rstudio-education.github.io/hopr">Hands-On Programming with R</a> - introductory book how to program in R, with hands-on examples, by Garrett Grolemund</p></li>
<li><p><a href="https://r4ds.had.co.nz/">R for Data Science</a> - a Tidyverse-oriented introduction to R online book, by Garrett Grolemund and Hadley Wickham</p></li>
<li><p><a href="https://adv-r.hadley.nz/">Advanced R</a> and <a href="https://advanced-r-solutions.rbind.io">Advanced R Solutions</a> - Advanced R programming, book and solutions, by Hadley Wickham and others</p></li>
<li><p><a href="https://rc2e.com">R Cookbook, 2nd Edition</a> - statistics-oriented R introduction, by James Long and Paul Teetor</p></li>
<li><p><a href="https://bookdown.org/csgillespie/efficientR/">Efficient R programming</a> - advanced concepts for efficient (R) programming, by Colin Gillespie &amp; Robin Lovelace</p></li>
<li><p><a href="https://rmarkdown.rstudio.com/lesson-1.html">RMarkdown from RStudio</a> - illustrated guide to RMarkdown</p></li>
<li><p><a href="https://rmd4sci.njtierney.com/">Rmarkdown for Scientists</a> book by Nicholas Tierney. <a href="https://github.com/njtierney/rmd4sci">GitHub</a></p></li>
<li><p><a href="https://mastering-shiny.org/">Mastering Shiny</a> book by Hadley Wickham. <a href="https://github.com/jcheng5/shiny-book">GitHub</a></p></li>
<li><p><a href="https://brshallo.github.io/r4ds_solutions/">Yet another ‘R for Data Science’ study guide</a>, by Bryan Shalloway. Tidyverse-oriented introduction to R</p></li>
<li><p><a href="https://togaware.com/onepager/">One Page R</a> book “Data Science Quick Start: Knowledge Discovery Through R” by Togaware, includes pdf slides and R code templates for various machine learning tasks</p></li>
<li><p><a href="https://rafalab.github.io/dsbook/">Data Analysis and Prediction Algorithms with R</a> book by Rafael Irizarry. Data science, statistics topics in R. <a href="https://github.com/rafalab/dsbook">GitHub</a>. <a href="https://genomicsclass.github.io/book/">Old version</a>, <a href="https://github.com/genomicsclass/labs">Labs</a>, and <a href="http://lorainelab.org/statistics-and-r-for-the-life-sciences/">Videos</a></p></li>
<li><p><a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual">R &amp; Bioconductor Manual</a> - One of the best R manual to brush up all major steps in data analysis/visualization. And links to other resources there. By Thomas Girke, UC Riverside</p></li>
<li><p><a href="https://github.com/geanders/RProgrammingForResearch">RProgrammingForResearch</a> - course notes for R Programming for Research. R learning from ground up to tidyverse, with lectures, homeworks, data, source files</p></li>
<li><p><a href="https://github.com/adw96/biostat561">biostat561</a> - Computational Skills for Biostatistics, from version control, R programming, ggplot, shiny, to Unix, LaTeX, Markdown, Python. By Amy Willis</p></li>
<li><p><a href="https://thriv.github.io/biodatasci2018/">THRIV datasci 2018</a> - THRIV Data Science course by Stephen Turner. Comprehensive coverage from R/RStudio introduction, RMarkdown, dplyr, ggplot2, shiny to all practical and statistical aspects of data cleaning, visualization, predictive modeling, survival analysis. <a href="https://stephenturner.github.io/workshops/index.html">Workshops</a></p></li>
<li><p><a href="https://datasciencelabs.github.io/">Introduction to Data Science</a> by Rafael Irizarry and Stephanie Hicks. The GitHub repo <a href="https://github.com/datasciencelabs/2020" class="uri">https://github.com/datasciencelabs/2020</a> has the latest course material, previous material for the course is available by changing the year number, e.g. <a href="https://github.com/datasciencelabs/2019" class="uri">https://github.com/datasciencelabs/2019</a>. <a href="https://github.com/datasciencelabs/data">Data for the course</a></p></li>
<li><p><a href="https://leanpub.com/rprogramming">R Programming for Data Science</a> by Roger Peng. Fundamentals of R programming and data science. <a href="https://github.com/rdpeng/rprogdatascience/">GitHub</a></p></li>
<li><p><a href="https://bookdown.org/rdpeng/RProgDA/">Mastering Software Development in R</a>, book by Roger D. Peng, Sean Kross, and Brooke Anderson. From R basics to package development</p></li>
<li><p><a href="https://rdpeng.github.io/Biostat776/">Statistical Computing</a> - Biostatistics 140.776 course by Roger Peng. <a href="https://www.youtube.com/c/RogerPeng/playlists">Youtube Playlists</a></p></li>
<li><p><a href="https://github.com/DataScienceSpecialization/courses">DataScienceSpecialization</a> - Course materials for the Data Science Specialization from JHU folks. Detailed lectures on each topic</p></li>
<li><p><a href="https://github.com/rafalab/ds4stats">ds4stats</a> - Data Science for Statisticians Workshop. Tidyverse, Data wrangling, visualization, ggplot2, machine learning. Rmds, lab exercises, links to ready-to-view lectures.</p></li>
<li><p><a href="https://github.com/rstudio-education/master-the-tidyverse/">master-the-tidyverse</a> - the Master the Tidyverse Workshop, <code>tidyverse</code>-oriented tutorials. <a href="https://github.com/rstudio-education/master-the-tidyverse-instructors">Instructor material</a></p></li>
<li><p><a href="https://moderndive.com/">Statistical Inference via Data Science, A ModernDive into R and the Tidyverse</a> - bookdown, statistics and data science using tidyverse. <a href="https://github.com/moderndive/moderndive_book">GitHub</a></p></li>
<li><p><a href="https://statsthinking21.org/">Statistical Thinking for the 21st Century</a> - bookdown, statistics theory illustrated with R. <a href="https://github.com/statsthinking21">GitHub material</a></p></li>
<li><p><a href="http://www.john-ros.com/Rcourse/">R (BGU course)</a> by Jonathan D. Rosenblatt. From R basics to regression, machine learning, graphics, shiny, advanced computing. <a href="https://github.com/johnros/RAMR-R-Course">GitHub</a></p></li>
<li><p><a href="https://github.com/rsquaredacademy-education/online-courses/">online-courses</a> - Free courses by RSquareAcademy. From data import to tidyverse, web scraping, regex, databases. Videos, slides, code, data.</p></li>
<li><p><a href="https://github.com/SISBID">SISBID</a> - several modules covering various aspects of data science. Summer Institute in Statistics for Big Data. Lectures, exercises, data. <a href="https://github.com/SISBID/Module1">Module 1, Big Data</a>, <a href="https://github.com/SISBID/Module2">Module 2, Visualization of Biomedical Big Data</a>, <a href="https://github.com/SISBID/Module3">Module 3, Reproducible research</a>, <a href="https://github.com/SISBID/Module4">Module 4</a>, <a href="https://github.com/SISBID/Module5">Module 5</a></p></li>
<li><p><a href="https://laderast.github.io/">Ted Laderas</a> and his <a href="https://ready4r.netlify.app/">Ready for R</a> course, the accompanying <a href="https://ready4r.netlify.app/labbook/">Ready for R: Notebook Reference</a> bookdown, and the <a href="https://r-bootcamp.netlify.app">R Bootcamp</a> tidyverse exercises</p></li>
<li><p><a href="https://geanders.github.io/RProgrammingForResearch/index.html">R Programming For Research</a> course by Brooke Anderson, Rachel Severson, and Nicholas Good, Colorado State University. Basic, intermediate, and advanced data analysis in R. <a href="https://github.com/geanders/RProgrammingForResearch">GitHub</a>, <a href="https://www.youtube.com/playlist?list=PLuGPtwgRXxqK1d-3V2MKCad24h-v75k8P">Youtube playlist</a> with short videos on R-related topics</p></li>
</ul>
</div>
<div id="rstudio" class="section level2">
<h2>RStudio</h2>
<ul>
<li><p><a href="https://rstudio.cloud">rstudio.cloud</a> - a web-based instance of R/RStudio. <a href="https://docs.google.com/presentation/d/1Lc0_hLgLUh0KSS5DIEZBUfE4KzeU8kH25nGY4w2kE1o/edit#slide=id.g606317e658_0_167">RStudio cloud cheatsheet and notes</a>, from <a href="https://twitter.com/lobrowR/status/1295114281428684806?s=20">Twitter</a> source. <a href="https://youtu.be/PviVimazpz8">RStudio Cloud for education - Mel Gregory</a> - 24 min video with all the essense of using RStudio cloud for education</p></li>
<li><p><a href="https://drdoane.com/my-rstudio-configuration/">Tips how to configure RStudio</a></p></li>
</ul>
</div>
<div id="datasets-in-r" class="section level2">
<h2>Datasets in R</h2>
<ul>
<li><p><code>library(help = "datasets")</code> - shows built-in R datasets</p></li>
<li><p>A list of over 1,000 datasets available in R packages, curated by <span class="citation">@VincentAB</span>. <a href="https://vincentarelbundock.github.io/Rdatasets/datasets.html" class="uri">https://vincentarelbundock.github.io/Rdatasets/datasets.html</a></p></li>
</ul>
<!--
- [Turn scripts into reproducible reports](http://kbroman.org/steps2rr/pages/reports.html) by Karl Broman 

- [R Markdown](http://kbroman.org/knitr_knutshell/pages/markdown.html) and [Knitr with R Markdown](http://kbroman.org/knitr_knutshell/pages/Rmarkdown.html) tutorials by Karl Broman

- [R Markdown - Dynamic Documents for R](https://support.rstudio.com/hc/en-us/articles/205368677-R-Markdown-Dynamic-Documents-for-R) from RStudio

- [R markdown cheat sheet](https://www.rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf) and [R Markdown Reference Guide](https://www.rstudio.com/wp-content/uploads/2015/03/rmarkdown-reference.pdf) from RStudio

- [RMarkdown from RStudio](https://rmarkdown.rstudio.com/lesson-1.html) - illustrated guide to RMarkdown 

- [Rmarkdown for Scientists](https://rmd4sci.njtierney.com/) book by Nicholas Tierney. [GitHub](https://github.com/njtierney/rmd4sci)

- [R Markdown: The Bigger Picture](https://github.com/garrettgman/rmarkdown-the-bigger-picture) slides for a talk at rstudio::conf 2019, by Garrett Grolemund. [Video](https://rstudio.com/resources/rstudioconf-2019/r-markdown-the-bigger-picture/), 19 min

- [Create beautiful and semantically meaningful articles with pandoc](https://pandoc-scholar.github.io), how to xreate scholar articles

- [Templates and tools for making a Résumé/CV, vitae R package](https://ropenscilabs.github.io/vitae/)
- [YAAC: Another Awesome CV is a template using Font Awesome and Adobe Source Font, LaTeX](https://github.com/darwiin/yaac-another-awesome-cv)
- RMarkdown CV built using the [pagedown](https://pagedown.rbind.io/) R package:
    - [Guangchuang Yu's CV](https://github.com/GuangchuangYu/cv)
    - [Nick Strayer's CV](https://github.com/nstrayer/cv)

- [Manubot](https://manubot.org/) - collaborative manuscripts in Markdown, flexible citations, Pandoc backend, Travis CI. https://manubot.org/
    - Himmelstein, Daniel S., Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, and Anthony Gitter. “[Open Collaborative Writing with Manubot](https://doi.org/10.1371/journal.pcbi.1007128).” PLoS Computational Biology 15, no. 6 (June 24, 2019)
-->
<!--
### Functions

- Resources by Karl Broman: [Transform repeated code into functions](http://kbroman.org/steps2rr/pages/functions.html), [Package functions for reuse](http://kbroman.org/steps2rr/pages/packages.html), [R package primer](http://kbroman.org/pkg_primer/)


### R packages

- [R package development - the Leek group way!](https://github.com/jtleek/rpackages) - Jeff Leek on developing R packages, links to resources

- [Writing an R package from scratch](https://hilaryparker.com/2014/04/29/writing-an-r-package-from-scratch/) tutorial by Hillary Parker

- [R packages](http://r-pkgs.had.co.nz/) book by Hadley Wickham

- [How to create an R package" example and links](https://github.com/isteves/r-pkg-intro) - R package example and links to resources

- [sinew](https://github.com/yonicd/sinew) - an R package for generating roxygen2 skeletons populated with information scraped from the function script

- [pRojects](https://github.com/lockedata/pRojects) - R package for making project templates

- [Package Development tutorial for useR! 2019 Toulouse](https://github.com/jennybc/pkg-dev-tutorial) by Jenny Bryant

- [Create A Package](https://kayla-morrell.github.io/CreateAPackage/) - Bioconductor 2020 workshop by Kayla Interdonato, Lori Shepherd

- [R package workshop](https://combine-australia.github.io/r-pkg-dev/) by COMBINE, an association for Australian students in bioinformatics,
-->
<!--
- [Manipulating, analyzing and exporting data with tidyverse](https://datacarpentry.org/R-ecology-lesson/03-dplyr.html) by Data Carpentry

- [Data Manipulation with dplyr](http://datascienceplus.com/data-manipulation-with-dplyr/) - a short blog post demonstrating dplyr's functionality, by Teja Kodali

- [Introduction to dplyr for Faster Data Manipulation in R] (https://rpubs.com/justmarkham/dplyr-tutorial) - another dplyr tutorial, with a 40 min [video](https://youtu.be/jWjqLW-u3hc)
-->
<!--
- [Visualisation of biomolecular data](https://lgatto.github.io/VisualisingBiomolecularData/index.html) by Laurent Gatto

- O'Donoghue, Sen I., Benedetta Frida Baldi, Susan J. Clark, Aaron E. Darling, James M. Hogan, Sandeep Kaur, Lena Maier-Hein, et al. "[Visualization of Biomedical Data](https://doi.org/10.1146/annurev-biodatasci-080917-013424)." Annual Review of Biomedical Data Science 1, no. 1 (July 20, 2018) - Visualization best practices (the use of length, area, color etc.)

- [Data visualization in R](https://datacarpentry.org/R-genomics/05-data-visualization.html) by Data Carpentry

- [Teaching materials for the R package ggplot2](https://github.com/jennybc/ggplot2-tutorial) by Jenny Bryant

- [Interactive plots in R](https://davetang.org/muse/2018/05/18/interactive-plots-in-r/) by Dave Tang

- [The R Graph Gallery, all graphs](http://www.r-graph-gallery.com/all-graphs/), visuals with code examples

- [Beautiful plotting in R: A ggplot2 cheatsheet](http://zevross.com/blog/2014/08/04/beautiful-plotting-in-r-a-ggplot2-cheatsheet-3/) - illustrated ggplot2 modifications
-->
<!--
## Best practices

- [Tips for organizing projects](http://kbroman.org/steps2rr/pages/organize.html), [Organizing data in spreadsheets](http://kbroman.org/dataorg/) by Karl Broman

- [Data Organization in Spreadsheets, common mistakes](http://www.datacarpentry.org/spreadsheet-ecology-lesson/02-common-mistakes/)

- [Software Carpentry reading material on software engineering and scientific computing](http://software-carpentry.org/reading/) 

- [Software development skills for data scientists](http://treycausey.com/software_dev_skills.html) by Trey Causey

- [ProjectTemplate](http://projecttemplate.net/) - an R package for advanced project management, [GitHub](https://github.com/johnmyleswhite/ProjectTemplate)

- Seemann, Torsten. “[Ten Recommendations for Creating Usable Bioinformatics Command Line Software](https://doi.org/10.1186/2047-217X-2-15).” GigaScience 2, no. 1 (December 2013)

- List, Markus, Peter Ebert, and Felipe Albrecht. “[Ten Simple Rules for Developing Usable Software in Computational Biology](https://doi.org/10.1371/journal.pcbi.1005265).” PLoS Computational Biology 13, no. 1 (January 2017)

- Taschuk, Morgan, and Greg Wilson. “[Ten Simple Rules for Making Research Software More Robust](https://doi.org/10.1371/journal.pcbi.1005412).” PLOS Computational Biology 13, no. 4 (April 13, 2017). [GitHub](https://github.com/oicr-gsi/robust-paper) 

- [Code and Data for the Social Sciences: A Practitioner’s Guide](https://web.stanford.edu/~gentzkow/research/CodeAndData.pdf) book by Matthew Gentzkow and Jesse Shapiro, PDF

- Wilson, Greg, D. A. Aruliah, C. Titus Brown, Neil P. Chue Hong, Matt Davis, Richard T. Guy, Steven H. D. Haddock, et al. "[Best Practices for Scientific Computing](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001745) ." PLoS Biology 2014

- Noble, William Stafford. “[A Quick Guide to Organizing Computational Biology Projects](https://doi.org/10.1371/journal.pcbi.1000424).” PLoS Computational Biology 5, no. 7 (July 2009) - Computational projects organization, folder structure, command line scripts, version control
-->
</div>

          </div>

          



          
        </div>

        <div class="body-footer">
          <p>Last updated on September 24, 2020</p>

          






  
  

<p class="edit-page">
  <a href="https://github.com/mdozmorov/BIOS524_biostatistical_computing_R/edit/master/content/reading/_index.html">
    <i class="fas fa-pen pr-2"></i>Edit this page
  </a>
</p>




          


          


        </div>

      </article>

      <footer>
    <hr>

    <div class="row course-info">
        <div class="col-md-7">
            <p>
                <strong>BIOS 524: Biostatistical Computing with R (Fall 2020)</strong><br>

                <a href="https://www.vcu.edu" target="_blank" rel="noopener">Virginia Commonwealth University</a> &emsp;&emsp;
                <a href="https://biostatistics.vcu.edu/" target="_blank" rel="noopener">Biostatistics Graduate Program</a>
            </p>

            <p>
                <a href="https://dozmorovlab.github.io" target="_blank" rel="noopener"><i class="fas fa-user"></i>
                    Dr. Mikhail Dozmorov</a> &emsp;&emsp;
                <a href="mailto:mdozmorov@vcu.edu"><i class="fas fa-envelope"></i>
                    mdozmorov@vcu.edu</a>
            </p>

            <p>
                <i class="far fa-calendar-alt"></i> Tuesday, Thursday &emsp;&emsp;
                <i class="far fa-clock"></i> 1:00pm – 2:20pm <br>
                <i class="fas fa-university"></i> Online
            </p>
        </div>

        <div class="col-md-5 credits">
            <p>All content licensed under a <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank" rel="noopener">Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License</a>.</p>
            
            <p>Content <i class="fab fa-creative-commons"></i> 2020 <a href="https://dozmorovlab.github.io" target="_blank" rel="noopener">MGD</a></p>
        
            <p>This site created with the <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> in <a href="https://bookdown.org/yihui/blogdown/" target="_blank" rel="noopener">blogdown</a> and <a href="https://gohugo.io/" target="_blank" rel="noopener">Hugo</a>. </p>
            
            <p><a href="https://github.com/mdozmorov/BIOS524_biostatistical_computing_R" target="_blank" rel="noopener"><i class="fab fa-github"></i> View the source at GitHub.</a></p>
        </div>
    </div>
</footer>


    </main>
  </div>
</div>


        

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js" integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/highlight.min.js" integrity="sha512-7t8APmYpzEsZP7CYoA7RfMPV9Bb+PJHa9x2WiUnDXZx3XHveuyWUtvNOexhkierl5flZ3tr92dP1mMS+SGlD+A==" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/languages/r.min.js"></script>
        
      

    

    
    

    
    
    <script>const code_highlighting = true;</script>
    

    
    
    <script>const isSiteThemeDark = false;</script>
    

    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/4.1.1/anchor.min.js" integrity="sha256-pB/deHc9CGfFpJRjC43imB29Rse8tak+5eXqntO94ck=" crossorigin="anonymous"></script>
    <script>
      anchors.add();
    </script>
    

    

    
    

    
    

    
    

    
    

    
    
    
    
    
    
    
    
    
    
    
    
    <script src="/js/academic.min.7c0855509a679f5c89322b27a9552a2f.js"></script>

    






    
    

    
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>

</html>
